src/main/java/codeu/model/data/Activity.java

```diff
@@ -0,0 +1,86 @@
+
+package codeu.model.data;
+
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.UserStore;
+
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.UUID;
+
+/**
+ * Class representing an activity (creating user, message, or conversation).
+ * Any creation done by a user prompts the creation of
+ * an activity object.
+ */
+public class Activity {
+  public final UUID id;
+  public final UUID owner;
+  public final String type;
```

Comment:
You could consider making `type` an `enum` since it seems like you intend for there to be a specific set of valid `type`s.

Reply:
Done